### ISSUE TITLE
Can you share the hyper-parameters for 6D/10D location finding tasks?

### ISSUE THREAD

**Issue by yfzhang114** [desi-ivanova/idad#2]
I use the default hyper-parameter setting but get the wrong result and the loss will not converge.
![image](https://user-images.githubusercontent.com/53161950/166211510-d9fde671-d508-4c27-8f6b-b114795bb240.png)

**Comment by desi-ivanova** [comment]
Hi, sure, here are the hyperparameters used (most are the default ones)

1. For 10D with InfoNCE bound:
```
python3 location_finding.py \
    --num-steps 100000 \
    --num-experiments 10 \
    --physical-dim 5 \
    --num-sources 2 \
    --lr 0.0004 \
    --num-experiments 10 \
    --encoding-dim 64 \
    --hidden-dim 512 \
    --mi-estimator InfoNCE \
    --device cuda
```
I just ran a shorter version of the above (`--num-steps 20000` which took ~40min on GPU, instead of 100K which will take 5x the time) and here are the results:

- EIG
<img width="281" alt="image" src="https://user-images.githubusercontent.com/16247687/166227625-dd905d08-569e-450b-af9b-ae5849b0e719.png">

- Loss curve
<img width="369" alt="image" src="https://user-images.githubusercontent.com/16247687/166228377-0483d264-5a2e-4630-8362-08c23f3ad92e.png">


2. For 6D with InfoNCE bound:
```
python3 location_finding.py \
    --num-steps 20000 \
    --num-experiments 10 \
    --physical-dim 3 \
    --num-sources 2 \
    --lr 0.0004 \
    --num-experiments 10 \
    --encoding-dim 64 \
    --hidden-dim 512 \
    --mi-estimator InfoNCE \
    --device cuda
```
Similarly, I ran a shorter version of the above (`--num-steps 20000` instead of 100K which will take 5x the time) and here are the results:

- EIG 
<img width="281" alt="image" src="https://user-images.githubusercontent.com/16247687/166227585-9ffa9a51-bfc4-4c67-b8a0-d4e7c2bbf79d.png">

- Loss curve
<img width="369" alt="image" src="https://user-images.githubusercontent.com/16247687/166227984-2440b610-f5d6-4202-827a-3186a4c2b861.png">

I hope this helps!
